src/docfx/Errors.cs

```diff
@@ -529,13 +529,24 @@ public static Error UidConflict(string uid, IEnumerable<string> conflicts = null
             return new Error(ErrorLevel.Warning, "uid-conflict", $"UID '{uid}' is defined in more than one file: {Join(conflicts)}");
         }
 
+        /// <summary>
+        /// Same uid defined within different versions with the different name.
+        /// Examples:
+        ///   - Same uid defined in multiple .md files with different versions have different titles.
+        /// </summary>
+        /// Behavior: ✔️ Message: ❌
+        public static Error UidPropertyConflict(string uid, string propertyName, IEnumerable<string> conflicts)
+        {
+            return new Error(ErrorLevel.Warning, "xref-property-conflict", $"UID '{uid}' is defined with different {propertyName}s: {Join(conflicts)}");
+        }
+
         /// <summary>
         /// Multiple articles with same uid contain overlapped monikers,
         /// and can't decide which article to use when referencing that uid with this overlapped version
         /// </summary>
         /// Behavior: ✔️ Message: ❌
         public static Error MonikerOverlapping(IEnumerable<string> overlappingmonikers)
-            => new Error(ErrorLevel.Error, "moniker-overlapping", $"Two or more documents have defined overlapping moniker: {Join(overlappingmonikers)}");
+            => new Error(ErrorLevel.Warning, "moniker-overlapping", $"Two or more documents have defined overlapping moniker: {Join(overlappingmonikers)}");
```

Comment:
>moniker-overlapping [](start = 46, length = 19)

since we changed the level and add new error code, do we need to confirm with PM?

Reply:
This follows the behavior of `uid-conflict`(warning) from v2, I would think this is fine

Reply:
@herohua